RocketChat/Rocket.Chat.ReactNative
# [IMPROVEMENT] Load only i18n files needed

## Proposed changes

Instead of loading all i18n files to memory (`require()`), loads only the ones really needed.

## Issue(s)	

Depends on https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/3011

## How to test or reproduce

- Set a new language and the app should apply it as usual
- Set a RTL language and the app should work as well
- Inspect i18n and check that only the current language (and 'en') is loaded to memory


## Screenshots

## Types of changes




## Checklist



## Further comments
